docs/the-guide/chapter-1-run-a-simple-ml-experiment/index.md

```diff
@@ -169,6 +169,16 @@ poetry install
 poetry shell
 ```
 
+!!! question
+
+	**Why Poetry?**
+
+	Poetry is a tool to manage Python dependencies. It is a more robust and
+	user-friendly alternative to `pip`. It is also more suitable for
+	reproducibility and collaboration by creating a lock file that can be used
+	to recreate the exact same environment.
```

Comment:
Do you think an illustration/example on why Poetry is so important and how pip can fail could be interesting? To make it clearer on the benefits of Poetry.